opentelemetry-api/src/opentelemetry/trace/status.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import enum
+import typing
+
+
+class StatusCanonicalCode(enum.Enum):
+    """Represents the canonical set of status codes of a finished Span."""
+
+    # pylint: disable=pointless-string-statement
+    """Not an error, returned on success.
+    """
```

Comment:
These are off by one in the generated docs since they're parsed as attribute docstrings. Since I had to make the change to check it anyway I put it in a PR for this branch: https://github.com/hectorhdzg/opentelemetry-python/pull/1.